shim.c

```diff
@@ -1810,9 +1810,6 @@ EFI_STATUS shim_verify (void *buffer, UINT32 size)
 	loader_is_participating = 1;
 	in_protocol = 1;
 
-	if (!secure_mode())
-		goto done;
-
```

Comment:
Why are you moving this check?  Doing so changes the shim_verify() function semantics since previously it would be a no-op if Secure Boot wasn't enabled.

Also, could you please add a commit message that explains the rationale for the change? Your subject line explains what the patch does but there isn't an explanation about why the change is needed.

Reply:
Yes I can move it to a separate commit. This move is necessary to ensure that measurements are being taken even if SecureBoot is disabled. I think this is actually more coherent with how load_image behaves as that also takes a measurement even if SecureBoot is disabled.